templates/portal/home.html.twig

```diff
@@ -660,41 +678,260 @@
+                    ,description: "You can send and receive secure communications with your care team staff through this system."|xl
+                    ,enabled: true
+                }
+                ,{
+                    title: "Health Snapshot"|xl
+                    ,icon: "clipboard"
+                    ,description: "See your immunization, medications, active prescriptions, allergy list, current problems list, and lab results"|xl
+                    ,enabled: true
+                }
+                ,{
+                    title: "Profile"|xl
+                    ,icon: "user"
+                    ,description: "Review and edit your medical profile information.  This includes your basic demographics (name, address, emergency contact). You can also review your insurance information if you use a third party insurer to help pay for treatment of care."|xl
+                    ,enabled: true
+                }
+                ,{
+                    title: "Billing Summary"|xl
+                    ,icon: "credit-card"
+                    ,description: ""|xl
+                    ,enabled: portal_two_ledger
+                }
+                ,{
+                    title: "Medical Reports"|xl
+                    ,icon: "gear"
+                    ,description: "Setup your digital signature for signing your clinical documents, update your login credentials, or change application settings in the portal."|xl
+                    ,enabled: ccdaOk or 'allow_custom_report' or portal_onsite_document_download
+                }
+                ,{
+                    title: "PRO Assessment"|xl
+                    ,icon: "flask-vial"
+                    ,description: "Questionnaires/Assessments using the Patient-Reported Outcomes Measurement Information System (PROMIS)."|xl
+                    ,enabled: isEasyPro
+                }
+                ,{
+                    title: "Settings"|xl
+                    ,icon: "gear"
+                    ,description: "Setup your digital signature for signing your clinical documents, update your login credentials, or change application settings in the portal."|xl
+                    ,enabled: true
+                }
+            ]
+        } %}
+
+        <!-- Help Card Reports List -->
+        {% include "portal/partial/_help_card.html.twig" with {
+            id: "help-card-reports-list"
+            ,backButton: true
+            ,backButtonCard: "reports-list-card"
+            ,backButtonClass: "reports-list-back-btn"
+            ,title: "Medical Reports Help"|xl
+            ,items: [
+                {
+                    title: "View Summary of Care"|xl
+                    ,icon: "notes-medical"
+                    ,description: "View your Summary of Care document that includes a printable version of your healthcare information.  This document can be used to transfer your care to another healthcare facility. In technical terms it is called a C-CDA."|xl
```

Comment:
don't use double spacing in translation constants

Reply:
also ensure this on other constants above and below

Reply:
Fixed